Assets/_BForBoss/_Core/Scripts/UserInterface/SettingsViewBehaviour.cs

```diff
@@ -57,7 +62,20 @@ private void Awake()
             _backButton.onClick.AddListener(ClosePanel);
             SetupViews();
         }
-        
+
+        private void OnValidate()
```

Comment:
As an aside, if we are cool with moving these kinda calls to `OnValidate()` then we should create a tech debt (backlog) task to refactor current classes to do the same. For consistency

Reply:
There aren't that many on awake currently? i don't think? so we can change them when we see them. Starting  now.